pageserver/src/task_mgr.rs

```diff
@@ -146,11 +145,10 @@ static TASKS: Lazy<Mutex<HashMap<u64, Arc<PageServerTask>>>> =
     Lazy::new(|| Mutex::new(HashMap::new()));
 
 task_local! {
-    // There is a Tokio watch channel for each task, which can be used to signal the
-    // task that it needs to shut down. This task local variable holds the receiving
-    // end of the channel. The sender is kept in the global registry, so that anyone
-    // can send the signal to request task shutdown.
-    static SHUTDOWN_RX: watch::Receiver<bool>;
+    // This is a cancellation token which will be cancelled when a task needs to shut down. This
+    // task local variable holds the receiving end of the channel. The sender is kept in the global
+    // registry, so that anyone can send the signal to request task shutdown.
```

Comment:
Trying to be quick, getting only 1/3 of correct-ish :)

```suggestion
    // This is a cancellation token which will be cancelled when a task needs to shut down. 
    // The root token is kept in the global registry, so that anyone can
    // send the signal to request task shutdown.
```

Reply:
Done.